doc/api/buffer.md

```diff
@@ -2159,6 +2159,8 @@ added: v0.1.90
 
 Decodes `buf` to a string according to the specified character encoding in
 `encoding`. `start` and `end` may be passed to decode only a subset of `buf`.
+If a byte sequence in the input is not valid in the given `encoding` then
+it is replaced with the replacement character `u+FFFD`.
```

Comment:
I think we use capital `U`:
```suggestion
it is replaced with the replacement character `U+FFFD`.
```

Reply:
wow, never used that GitHub feature before. Do I need to squash commits, or is this done by some machinery anyway?

Reply:
Whoever merges this will squash the commits and take care of any necessary other changes (e.g. `doc:` in the commit message rather than `docs:`)

Reply:
Lovely. Thanks for your help @addaleax and everyone -- that's a nice first-contributor experience.